pkg/apb/types.go

```diff
@@ -274,9 +275,38 @@ func (si *ServiceInstance) RemoveBinding(bindingUUID uuid.UUID) {
 
 // BindInstance - Binding Instance describes a completed binding
 type BindInstance struct {
-	ID         uuid.UUID   `json:"id"`
-	ServiceID  uuid.UUID   `json:"service_id"`
-	Parameters *Parameters `json:"parameters"`
+	ID           uuid.UUID   `json:"id"`
+	ServiceID    uuid.UUID   `json:"service_id"`
+	Parameters   *Parameters `json:"parameters"`
+	CreateJobKey string
+}
+
+// UserParameters - returns the Parameters field with any keys and values
+// removed that are typically added by the broker itself. The return value
+// should represent what a OSB API client provides in a bind request.
+func (bi *BindInstance) UserParameters() Parameters {
+	if bi.Parameters == nil {
+		return nil
+	}
+	userparams := make(Parameters)
+	for key, value := range *bi.Parameters {
+		switch key {
+		// Do not copy keys that are generally added by the broker itself.
+		case "cluster", "namespace", "_apb_provision_creds":
```

Comment:
One day we should make these constants. Possibly outside the scope of this PR but I think that this is the second place we use those values.